english/chillmusic/chillmusic.md

```diff
@@ -8,5 +8,5 @@ Click to enjoy the soothing tones of [Rick Astley, Never gonna give you up](http
 
 Click to listen to a nice EDM track:
 [Faded - Alan Walker - Lost Stories Mix](https://soundcloud.com/loststories/alan-walker-faded-lost-stories-remix)
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.